spec/requests/citizens/check_answers_spec.rb

```diff
@@ -60,7 +60,7 @@
 
     it 'displays the name of the firm' do
       subject
-      expect(response.body).to include(firm.name)
+      expect(response.body).to include(HTMLEntities.new.encode(firm.name))
```

Comment:
This doesn't seem to be related to this PR. Has it crept in from another change?

Reply:
See above